samples/stock/demo/orderbook-chart/demo.js

```diff
@@ -0,0 +1,185 @@
+function getRandomNumber(min, max) {
+    return Math.round(Math.random() * (max - min)) + min;
+}
+
+function generateBidAndAskData(n) {
+    const data = [[], []];
+    let bidPrice = getRandomNumber(29000, 30000),
+        askPrice = bidPrice + 0.5;
+    for (let i = 0; i < n; i++) {
+        bidPrice -= ((i * getRandomNumber(8, 10)));
+        askPrice += ((i * getRandomNumber(8, 10)));
+        data[0].push({
+            x: i,
+            y: (-i - 1) * getRandomNumber(70000, 110000),
+            price: bidPrice
+        });
+
+        data[1].push({
+            x: i,
+            y: (i + 1) * getRandomNumber(70000, 110000),
+            price: askPrice
+        });
+    }
+    return data;
+}
+
+const [bidsData, asksData] = generateBidAndAskData(10);
+
+function updateData(series) {
+    const data = generateBidAndAskData(10);
+    series.forEach((s, i) => {
+        s.setData(data[i]);
+    });
+}
+
+Highcharts.chart('container', {
+    chart: {
+        type: 'bar',
+        backgroundColor: '#23232f',
+        marginTop: 70,
+        events: {
+            load() {
+                this.titles = [];
+                setInterval(() => {
+                    updateData(this.series);
+                }, 200);
```

Comment:
If the interval is below 1s, then shouldn't we also decrease the animation time? 🤔